openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -27320,6 +27320,95 @@ OS:Generator:FuelSupply,
       \minimum 0.0
       \maximum 1.0
 
+OS:Generator:PVWatts,
+      \min-fields 4
+      \memo Describes a simple set of inputs for an array of photovoltaic (PV) modules as described
+      \memo in the PVWatts software. A series of different PVWatts arrays can be connected to a
+      \memo single electric load center (preferably through an
+      \memo ElectricLoadCenter:Inverter:PVWatts). Array tilt and azimuth can be either specified
+      \memo on this object or taken from a referenced building surface or shading surface. If
+      \memo a surface is specified, the array participates normally in all shading calculations.
+  A1, \field Handle
+       \type handle
+       \required-field
+  A2, \field Name
+      \required-field
+      \reference GeneratorNames
+  A3, \field PVWatts Version
+      \type choice
+      \key 5
+  N1, \field DC System Capacity
+      \note Nameplate rated DC system capacity in watts
+      \type real
+      \required-field
+      \units W
+      \minimum> 0
+  A4, \field Module Type
+      \type choice
+      \key Standard
+      \key Premium
+      \key ThinFilm
+      \default Standard
+  A5, \field Array Type
+      \type choice
+      \key FixedOpenRack
+      \key FixedRoofMounted
+      \key OneAxis
+      \key OneAxisBacktracking
+      \key TwoAxis
+      \default FixedOpenRack
+  N2, \field System Losses
+      \type real
+      \default 0.14
+      \minimum 0
+      \maximum 1
+  N3, \field Tilt Angle
+      \note The tilt angle is the angle from horizontal of the photovoltaic modules in the array.
+      \type real
+      \units deg
+      \default 20
+      \minimum 0
+      \maximum 90
+  N4, \field Azimuth Angle
+      \note For a fixed array, the azimuth angle is the angle clockwise from true north describing
+      \note the direction that the array faces. For an array with one-axis tracking, the azimuth
+      \note angle is the angle clockwise from true north of the axis of rotation.
+      \type real
+      \units deg
+      \default 180
+      \minimum 0
+      \maximum< 360
+  A6, \field Surface Name
+      \type object-list
+      \object-list AllShadingAndHTSurfNames
+  N5; \field Ground Coverage Ratio
+      \note Applies only to arrays with one-axis tracking and is the ratio of module surface area
+      \note to area of the ground or roof occupied by the array.
+      \type real
+      \default 0.4
+      \minimum 0
+      \maximum 1
+
+OS:ElectricLoadCenter:Inverter:PVWatts,
+      \min-fields 2
+      \memo Electric power inverter to convert from direct current (DC) to alternating current
+      \memo (AC) in an electric load center that contains Generator:PVWatts objects.
+      \memo It implements the PVWatts inverter performance curves.
+  A1, \field Handle
+       \type handle
+       \required-field 
+  A2, \field Name
+      \reference InverterList
```

Comment:
Should be required right?

Reply:
Oops. Looks like the other inverters have their Name field as required. So yes should be.